search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/query/TypedSearchQueryTransformer.java

```diff
@@ -77,15 +75,12 @@ private SearchSourceConfig toSearchSourceConfig(final QueryResultConfig resultCo
     return resultConfigTransformer.apply(resultConfig);
   }
 
-  private SearchQuery toSearchQuery(final F filter, final SearchQuery authCheck) {
-    final var filterTransformer = getFilterTransformer(filter.getClass());
-    final var transformedQuery = filterTransformer.apply(filter);
-    return and(transformedQuery, authCheck);
+  private SearchQuery toSearchQuery(final F filter) {
+    return getFilterTransformer(filter).apply(filter);
   }
 
   private List<String> toIndices(final F filter) {
-    final var filterTransformer = getFilterTransformer(filter.getClass());
-    return filterTransformer.toIndices(filter);
+    return List.of(getFilterTransformer(filter).toIndex(filter).getAlias());
```

Comment:
❓ do we have aliases for all indices?

Reply:
Yes, and this is a good practice so I don't think it's accidental.  
```
curl -s http://localhost:9200/_cat/aliases | sort
```